### ISSUE TITLE
Ability to create groups of planets

### ISSUE THREAD

**Issue by RedOrion** [plainblack/Lacuna-Server-Open#400]

**Comment by dmcbride** [comment]
I can add more information to the empire info (star ID/name - already adding the x/y co-ords) while I'm doing the sitter stuff, but then having some sort of sort ability in the UI is a whole different story :)

**Comment by dmcbride** [comment]
Sorting in the UI should be doable by: user (default - my colonies, my stations, allied stations, each baby), star, zone.  For the latter two, a colour-coding for each entry might come in handy so you can see at a glance whose body each is, or at least the primary four (my colony, my station, allied station, some baby's colony).

Each colony needs a pop-up as well which can describe the colony in minor details (category, real owner, zone, x,y location) as well as options (move to its system in the star map, switch to it in the planet map, etc.).

**Comment by Imzogelmo** [comment]
I like both of the things you guys are saying, but I think it's important to point out that you are saying 2 very different things-- one is a user-defined grouping and the other is an organic one based on real criteria such as location or ownership.

Certainly it would be helpful to have either one when you have a lot of planets/stations to check on.

**Comment by dmcbride** [comment]
yes, they are different.  However, I think the first step in "order by anything, really" is to have "anything" as part of the request, so I've done that now (not integrated yet).  The only way to go further beyond this is either a) local storage (pure javascript) or b) some sort of user-to-planet tagging system.  Which isn't a horrible idea, but may take a fair bit more work.  So, I'd like to start by focusing on what we think might be popular sorting systems first, and move towards fully-generic ones later, if it's still something people really really want.  As it is, it'll already be miles ahead of what we have now.

**Comment by Imzogelmo** [comment]
I don't know what would be popular, as I only have one data point (mine) to judge by, but "host star", "zone", and "owner" (for when the sitter changes occur) seem to me the most useful ways to sort. Naturally, all of the same star will also be in the same zone, which makes a 2-tier tree right off. The last one, owner, is clearly useful not only as a way to sort but also a way to filter. 

Actually, UF had this idea and I thought it may be a quick, poor-man's way to mark them: Just append the zone on to the end of each body's name in the UI list. If all you need is a nearby colony to pick up a trade, that would accomplish the same thing and probably be relatively cheap to do.

**Comment by Wolfsfriend** [comment]
As a player I like this Idea. It would be nice if I could have all the colonies in one zone grouped together, then another zone grouped etc... It would make working between planets easier to do, especially when you get a lot of planets or stations.
